shared-lib/lib/Expression/ExpressionFunctions.js

```diff
@@ -40,6 +40,9 @@ export const ExpressionFunctions = {
 	split: (str, separator) => {
 		return (str + '').split(separator)
 	},
+	join: (arr = [], separator = ',') => {
+		return (arr.constructor === Array ? arr : [arr]).join(separator)
```

Comment:
I think this could use `Array.isArray(arr)` to check if it is an array

Reply:
This sounded like it thought the equality was faster, but I'm happy to switch for readability if you'd prefer?
https://stackoverflow.com/a/26633883

Reply:
I'm not sure that we should be worried about speed here, these expression executor is going to be rather suboptimal by design (its an interpreter running inside an interpreter/jit)
But I don't feel that strongly about how this should be done. you thought this was better, which is enough for me

Reply:
Heh very true. Maybe readability should have won out, feel free to just push a change. There didn't seem to be much else in it compatibility wise etc.